CMakeLists.txt

```diff
@@ -1050,6 +1052,11 @@ if(openPMD_BUILD_TESTING)
                 endforeach()
             endif()
         endif()
+        if(openPMD_HAVE_ADIOS2)
```

Comment:
If the issue is only in execution, you could skip the `add_test` here for Windows or try to search for `sh` and skip it conditionally if the command is not found:
https://github.com/ECP-WarpX/WarpX/blob/21.01/cmake/WarpXFunctions.cmake#L1-L12

Reply:
`sh` on Windows is no issue actually. I didn't expect it, but it seems to work.